activiti-cloud-service-common-dependencies/pom.xml

```diff
@@ -42,6 +42,11 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.activiti.cloud.common</groupId>
+        <artifactId>activiti-cloud-service-common-properties</artifactId>
```

Comment:
@salaboy I wanted to get your feedback concerning names before merging this. The idea of this module is to provide default values for properties that are common (same name) in several services (runtime-bundle, query, audit). Currently it has only the property to enable/disable clean up endpoints in the REST API, but we can image that it will have other properties in the future such as enable/disable REST API.

Reply:
@erdemedeiros instead of common-properties I would called them common-config, because we have things like -> @Configuration
And common configurations should have their own module.

Reply:
@salaboy I agree with you on renaming it to `common-config`. It sounds better.

>And common configurations should have their own module.

Not sure I got this part correctly. @miguelruizdev is adding it as new Maven module, is that what you meant?